README.md

```diff
@@ -58,10 +68,12 @@ You can access to applications as follow:
 To stop all
 
 ```sh
-make stop
+make stop-dev
 ```
 
-## Develop packages
+## Develop without Docker
```

Comment:
I'm not sure that this is necessary to support Dockerless development env. Developers that join the project are more likely to use the Docker version of the dev environment since it is plug and play

Reply:
Since the development environment with Docker is not fully functional (admin app is not working correctly for now), maybe we can keep it ?